packages/kuma-gui/src/test-support/mocks/src/mesh-insights/_.ts

```diff
@@ -41,11 +40,9 @@ export default ({ env, fake }: EndpointDependencies): MockResponder => (req) =>
         gatewayDelegated,
       },
       policies: {
-        ...policyTypes.reduce((prev: Record<string, { total: number }>, item) => {
-          if (fake.datatype.boolean()) {
-            prev[item] = {
-              total: fake.number.int(20),
-            }
+        ...fake.kuma.policyNames().reduce((prev: Record<string, { total: number }>, item) => {
+          prev[item] = {
+            total: fake.number.int(20),
```

Comment:
This was the bit of code that led me to do all this. I wanted to remove the boolean here and just use `policyNames` but I couldn't get it to give me a random amount of names.